lib/pbench/cli/agent/commands/tools/base.py

```diff
@@ -0,0 +1,37 @@
+from pbench.agent.base import BaseCommand
+
+
+class ToolCommand(BaseCommand):
+    """Common Tool Class"""
+
+    def __init__(self, context):
+        super(ToolCommand, self).__init__(context)
+
+    def remote(self, path):
+        """List all remotes in a given path"""
+        return sorted([p.name for p in path.iterdir() if p.name != "__trigger__"])
+
+    def tools(self, path):
+        """List all tools in a given path"""
+        return sorted(
+            [
+                p.name
+                for p in path.iterdir()
+                if p.name != "__label__" and p.suffix != ".__noinstall__"
+            ]
+        )
+
+    def tools(self, path):
+        """List all tools in a given path"""
+        return [
+            p.name
+            for p in path.iterdir()
+            if p.name != "__label__" and p.suffix != ".__noinstall__"
+        ]
```

Comment:
So... two identical method signatures with slightly different implementations? I think this is also your Travis complaint.

Reply:
fixed